infrastructure/variables.tf

```diff
@@ -66,6 +66,7 @@ variable "ccd_gateway_url" {
 }
 
 variable "component" {
+  default = "repl-docomosis-service"
```

Comment:
This is mis-spelt and it needs to be repl-docmosis-backend.

See component defined here https://github.com/hmcts/ethos-repl-docmosis-service/blob/master/Jenkinsfile_CNP

Reply:
Refactored